### DISCUSSION TITLE
How can I go to previous/next class in python?

### DISCUSSION THREAD

**Discussion by karambaq** [JetBrains/ideavim#673] [Q&A]
I wonder how can I do those things:

1) Visual select current method
2) Go to previous/next class in Python

**Comment by citizenmatt** [comment]
You should be able to use `[m` and `]m` to move to the next/previous method. Selecting the current method could be achieved using the IDE's own "Extend Selection" action and shortcuts, or you can look at the emulation of the [vim-indent-object](https://github.com/michaeljsmith/vim-indent-object) text object, which you can enable in your `~/.ideavimrc`:

```
Plug 'michaeljsmith/vim-indent-object'
```
